### PR TITLE
feat: show update date on report and creation and update date in proj…

### PR DESCRIPTION
**Pull request by Sparkier**
…ect settings

# Description

fix ZEN-307

image
image

### CODE REVIEW

File: backend/pyproject.toml [hunk 1]
```diff
@@ -42,8 +42,6 @@ include = ["zeno_backend"]
 
 [tool.ruff]
 select = ["E", "F", "I", "PD", "N", "D", "T20"]
-# Ignore conflicting specs between linter and formatter. Formatter will catch line length.
-ignore = ["D206", "E501"]
```
Comment (Sparkier) [inline backend/pyproject.toml:46]: This has to be removed otherwise the CI does not catch things that should be caught (e.g. long lines)

**Comment by cabreraalex** [comment]
Should we show the time too? It then gets iffy on time zones but might be useful to see time.

**Comment by Sparkier** [comment]
> Should we show the time too? It then gets iffy on time zones but might be useful to see time.

Maybe in the settings, I would not on the report page, not really relevant there, is it?

**Comment by cabreraalex** [comment]
Can collaborators see the settings? asking for the class assignment for the profs to see the time maybe.

File: frontend/src/lib/components/popups/ProjectPopup.svelte [hunk 2]
```diff
@@ -109,6 +109,8 @@
 						<Checkbox checked={config.public} on:click={() => (config.public = !config.public)} />
 						<span>Public visibility</span>
 					</div>
+					<span>Created at: {new Date(config.createdAt).toLocaleDateString()}</span>
```
Comment (cabreraalex) [inline frontend/src/lib/components/popups/ProjectPopup.svelte:112]: I think just `Created` and `Updated` is fine

File: frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte [hunk 3]
```diff
@@ -91,11 +91,13 @@
 				report
 			/>
 		</div>
-		<h5 class="mt-4 ml-1 text-lg">Author: {data.report.ownerName}</h5>
-		<hr class="mt-4 text-grey-light" />
+		<h5 class="mt-4 text-lg">Author: {data.report.ownerName}</h5>
+		<span class="text-grey-darker mt-2">{new Date(data.report.updatedAt).toLocaleDateString()}</span
```
Comment (cabreraalex) [inline frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte:95]: I would add `Updated` before the date here. no colon needed just `Update 10/10/2023 at 12:23`

**Review by cabreraalex — CHANGES_REQUESTED** [review]
Do we want to show this on projects too?

**Comment by Sparkier** [comment]
> Do we want to show this on projects too?

I don't think it's as important for project and I'm not sure where. But could show as well no strong opinion.

**Comment by cabreraalex** [comment]
Fine no for no

File: frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte (hunk 3 above)
Comment (Sparkier) [reply frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte:95]: Actually, maybe fine with time generally:
image

**Review by cabreraalex — APPROVED** [review]

**Pull request merged**